jcl/src/java.base/share/classes/java/lang/invoke/StaticFieldVarHandle.java

```diff
@@ -90,9 +93,17 @@
 				
 		return populateMHs(operationsClass, lookupTypes, exactTypes);
 	}
+	
+	private static final JITHelpers jitHelpers = JITHelpers.getHelpers();
```

Comment:
I think this code can all be removed now.

Reply:
Done.